extensions/amp-story/0.1/amp-story-bookend.js

```diff
@@ -56,6 +57,9 @@ const FULLBLEED_CLASSNAME = 'i-amphtml-story-bookend-fullbleed';
 /** @private @const {string} */
 const HIDDEN_CLASSNAME = 'i-amphtml-hidden';
 
+// TODO(#14591): Clean when older version is deprecated.
+const BOOKEND_VERSION_2 = 'v2.0';
+const BOOKEND_VERSION = 'bookend-version';
```

Comment:
nit: one of these refers to a key and one refers to a value, but the names don't make that clear